### PR TITLE
update spanish translation

### PR DESCRIPTION
**Pull request by greenfoo**

### CODE REVIEW

File: resources/language/resource.language.es_es/strings.po [hunk 1]
```diff
@@ -105,6 +123,10 @@ msgctxt "#30061"
 msgid "{0.title}\nAuthor: {0.author}\nVideos: {0.videoCount}"
 msgstr "{0.title}\nAutor: {0.author}\nVídeos: {0.videoCount}"
 
+msgctxt "#30062"
+msgid "Nueva búsqueda"
```
Comment (lekma) [inline resources/language/resource.language.es_es/strings.po:127]: "New search" missing? Is it a problem (I'm not too familiar with this localization system)?

File: resources/language/resource.language.es_es/strings.po [hunk 2]
```diff
@@ -135,3 +157,19 @@ msgctxt "#30105"
 msgid "Publicly available instances"
 msgstr "Instancias públicas"
 
+msgctxt "#30111"
+msgid "Feed management"
```
Comment (lekma) [inline resources/language/resource.language.es_es/strings.po:161]: shouldn't this be "Persistent data"? (maybe pull again from master an retry?)

**Comment by greenfoo** [comment]
Ups... sorry about that. The "rebase" did not take care of the reuse ID and it got mixed up.
Should be OK now.

**Comment by lekma** [comment]
thanks for the help

**Pull request merged**